R/as_polars.R

```diff
@@ -0,0 +1,109 @@
+#' To polars DataFrame
+#'
+#' [as_polars_df()] is a generic function that converts an R object to a
+#' polars DataFrame. It is basically a wrapper for [pl$DataFrame()][pl_DataFrame],
+#' but has special implementations for Apache Arrow-based objects such as
+#' polars [LazyFrame][LazyFrame_class] and [arrow::Table].
+#'
+#' For [LazyFrame][LazyFrame_class] objects, this function is a shortcut for
+#' [$collect()][LazyFrame_collect] or [$fetch()][LazyFrame_fetch], depending on
+#' whether the number of rows to fetch is infinite or not.
+#' @rdname as_polars_df
+#' @param x Object to convert to a polars DataFrame.
+#' @param ... Additional arguments passed to methods.
+#' @examples
+#' # Convert a data.frame to a polars LazyFrame
+#' lf = as_polars_df(mtcars)$lazy()
```

Comment:
it would probably have more interest to show the example with the ArrowTable because for classic data.frames `pl$DataFrame()` works fine